updater.js

```diff
@@ -48,12 +56,39 @@ class AlAzureUpdater {
             return new ApplicationTokenCredentials(this._clientId, this._domain, this._clientSecret);
         }
     }
-    
+
     syncWebApp(callback) {
         const credentials = this._getAzureCredentials();
         const webSiteClient = new WebSiteManagementClient(credentials, this._subscriptionId);
         return webSiteClient.webApps.syncRepository(this._resourceGroup, this._webAppName, callback);
     }
+
+    setEnvConfigChanges(envObject, callback) {
+        var updateEnv = envObject;
+        m_util.updateAppSettings(updateEnv, this.azureClientObject, function (settingsError) {
+            if (settingsError) {
+                return callback(settingsError);
+            } else {
+                return callback(null);
+            }
+        });
+    }
+    run(envObject, callback) {
```

Comment:
I suggest we don't pass `envObject` as parameter. But rather have updater reading it itself.
```
const envObject = fs:readFile(process.env.azure_fun_update_config_name);
```

And have `AZURE_FUN_UPDATE_CONFIG_NAME = ../al-ehub-collector.json` env var set in the [template](https://github.com/alertlogic/ehub-collector/blob/v1/templates/ehub.json#L121-L158)